.github/workflows/tests-postgres.yml

```
@@ -0,0 +1,50 @@
+name: Go
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  postgres:
+    strategy:
+      matrix:
+        dbversion: ["postgres:latest"]
+        go: ["1.20", "1.19", "1.18"]
```

This can probably do with updating
```suggestion
        go: ["1.21", "1.20", "1.19"]
```

---

ty!